lightning/src/util/events.rs

```diff
@@ -340,6 +349,13 @@ impl Writeable for Event {
 					(2, reason, required)
 				});
 			},
+			&Event::DiscardFunding { ref channel_id, ref transaction } => {
+				12u8.write(writer)?;
```

Comment:
This should use an odd type - our even types means "if you see this, and dont know what it is, fail", but we'd ideally like old LDK versions to simply ignore the new event.